content/ko/self-healing.md

```diff
@@ -0,0 +1,9 @@
+---
+title: 셀프 힐링 (Self Healings)
+status: Completed
+category: 속성
+tags: ["인프라스트럭처", "아키텍처", ""]
+---
+
+셀프 힐링(Self Healing)은 하드웨어 또는 소프트웨어의 특정 오류 또는 실패로부터 사용자의 개입 없이 자동적으로 복구할 수 있는 시스템의 고유한 능력을 가리킨다. 이는 "수렴" 혹은 "제어" 루프라는 메커니즘을 통해 이루어지며, 시스템의 현 상태를 지속적으로 모니터링하고 초기에 운영자가 설정한 목표 상태와 비교한다. 
+만약 두 상태간에 이질성이 발견되면, 시스템은 자동으로 조정 조치를 취하게 된다.
```

Comment:
[원문]
> A self-healing system is capable of recovering from certain types of failure without any human intervention. 
It has a "convergence" or "control" loop that actively looks at the system’s actual state and 
compares it to the state that the operators initially desired. 
If there is a difference (e.g., fewer application instances are running than desired), 
it will take corrective action (e.g., start new instances).

위의 @seokho-son 님의 코멘트에 있는 semantic line breaks 관련하여, 다음과 같이 줄바꿈을 제안해 봅니다. ㅎㅎ

그리고 원문 반영을 좀 더 해 보았습니다.
('하드웨어 또는 소프트웨어의' 는 원문에 없지만 추가하신 것으로 보이네요. 저는 독자의 이해를 돕는 데 좋아 보입니다.. ㅎㅎ)

```suggestion
셀프 힐링(Self Healing)은 하드웨어 또는 소프트웨어의 특정 오류 또는 실패로부터 
사용자의 개입 없이 자동적으로 복구할 수 있는 시스템의 고유한 능력을 가리킨다. 
이는 "수렴" 혹은 "제어" 루프라는 메커니즘을 통해 이루어지며, 
시스템의 현 상태를 지속적으로 모니터링하고 초기에 운영자가 설정한 목표 상태와 비교한다. 
만약 두 상태간에 이질성이 발견되면(예: 목표보다 적은 수의 애플리케이션 인스턴스가 실행 중),
시스템은 자동으로 조정 조치를 취하게 된다(예: 새로운 인스턴스 실행).
```